frontend/src/metabase/ui/components/inputs/TimeInput/TimeInput.styled.tsx

```diff
@@ -6,5 +6,11 @@ export const getTimeInputOverrides =
       defaultProps: {
         size: "md",
       },
+      styles: _theme => ({
```

Comment:
just `() =>`?

Reply:
that would do, I just thought leaving `_theme` was a good idea, so we know what props are being passed. But we can literally see a lot of examples from other `*.styled.tsx` inside `ui` folder anyway.